docs/index.md

```diff
@@ -5,13 +5,8 @@ SleepECG provides tools for sleep stage classification when [EEG](https://en.wik
 - detecting heartbeats from ECG signals, and
 - classifying sleep stages (which includes preprocessing, feature extraction, and classification).
 
-## Overview
-- [Datasets](./datasets) shows all available datasets and instructions for retrieving NSRR data.
-- [Heartbeat Detection](./heartbeat_detection) demonstrates how to use the included heartbeat detector and shows benchmark results.
-- [Feature Extraction](./feature_extraction) lists all implemented HRV features.
-- [Classification](./classification) describes the included sleep stage classifiers and how to use the classification API.
-- [Configuration](./configuration) explains configuration settings in SleepECG.
-- [API](./api) has detailed information about all public functions and classes in SleepECG.
+## Documentation
+Documentation for SleepECG is available on [Read the Docs](https://sleepecg.readthedocs.io/en/stable/index.html).
```

Comment:
I think this section isn't necessary on the docs' landing page.

Reply:
True. The reason I added it and at the same time removed the Overview (your next comment) is that the contents of `index.md` is identical to `README.md`. Which is easier to maintain, otherwise they might get out of sync quickly.

Reply:
Ah I see! Then we could use `README.md` directly to avoid having to sync them manually?

Reply:
Maybe... but how? Also, I'm not sure I want to include the badges in our docs, those should only be used in README.md. I think the simplest solution is to sync them manually (we can add a note in our CONTRIBUTING.md).

Reply:
I was hoping replacing `- Home: index.md` with `- Home: ../README.md` in `mkdocs.yml` would do the trick, but apparently that doesn't work, so nevermind 🤷.

Reply:
OK, so let's sync them manually then.